### PR TITLE
Escape > as well as < in HTML for compatibility with JDK 8

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/maven/plugins/hpi/TagLibInterfaceGeneratorMojo.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public boolean accept(File dir, String name) {
 
                         JDocComment javadoc = m.javadoc();
                         if (doc!=null)
-                            javadoc.append(doc.getText().replace("&","&amp;").replace("<","&lt;"));
+                            javadoc.append(doc.getText().replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;"));
```
Comment (KostyaSha) [inline src/main/java/org/jenkinsci/maven/plugins/hpi/TagLibInterfaceGeneratorMojo.java]: No good escaper in EscapeUtils?

Comment (jglick) [inline src/main/java/org/jenkinsci/maven/plugins/hpi/TagLibInterfaceGeneratorMojo.java]: Maybe, I did not look. This suffices.

**Comment by jglick** [comment]
Feel free to file a more elaborate change that would cover more (hypothetical) cases. I am making the minimal change which allows core Javadoc generation to pass.

**Comment by KostyaSha** [comment]
AFAIK javadoc is html, so why not escape it with html escaper?

**Comment by jglick** [comment]
Well the test is that the core build passes!

**Pull request merged**